src/main/java/frc/robot/subsystems/gpm/Shooter.java

```diff
@@ -273,7 +269,9 @@ public void setTargetVelocity(double output) {
 	 * @return boolean indicating whether both PIDs are at their setpoints
 	 */
 	public boolean atSetpoint() {
-		return EqualsUtil.epsilonEquals(getLeftMotorRPM(),leftPID.getSetpoint(),TOLERANCE);
+		// TODO: pid.atSetpoint should work
+		return EqualsUtil.epsilonEquals(getLeftMotorRPM(),leftPID.getSetpoint(),TOLERANCE)&&
+		EqualsUtil.epsilonEquals(getRightMotorRPM(),rightPID.getSetpoint(),TOLERANCE);
 		//return leftPID.atSetpoint() && rightPID.atSetpoint();
 	}
```

Comment:
Use pid.atSetpoint(). Do not recreate the wheel.

Reply:
It wasn't working for some reason and we didn't have time to debug. That's why there's a TODO.